R/protein_quant.R

```diff
@@ -499,15 +468,19 @@ pquant <- function (pepData, combine_fn) {
   pro_id <- attr(pepData, "cnames")$emeta_cname
 
   # Quantitate the heck out of the peptides!
-  res <- merge(x = pepData$e_meta[, c(pep_id, pro_id)],
-               y = pepData$e_data,
-               by = pep_id,
-               all.x = FALSE,
-               all.y = TRUE) %>%
-    dplyr::select(-rlang::sym(pep_id)) %>%
-    dplyr::group_by(!!rlang::sym(pro_id)) %>%
-    dplyr::mutate(dplyr::across(.cols = -dplyr::any_of(pro_id),
-                                .fns = combine_fn)) %>%
+  res <- merge(
```

Comment:
Maybe we should swap this to a join since everything else is dplyr too

Reply:
I think if merge works we can keep it, the dplyr changes were to get rid of reshape2 in the dependencies.

Reply:
Might also make a note that we should think about removing data.table